packages/storybook-utils/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@web/storybook-utils",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Utils for Storybook",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/modernweb-dev/web.git",
+    "directory": "packages/storybook-utils"
+  },
+  "author": "modern-web",
+  "homepage": "https://github.com/modernweb-dev/web/tree/master/packages/storybook-utils",
+  "type": "module",
+  "exports": {
+    ".": {
+      "default": "./index.js",
+      "types": "./index.d.ts"
```

Comment:
types should be the first key, before "default"

https://publint.dev/rules#exports_types_should_be_first

Reply:
thanks!